packages/build/src/plugins/expected_version.js

```diff
@@ -60,4 +55,21 @@ const isAutoPlugin = function ({ loadedFrom }) {
   return loadedFrom === 'auto_install'
 }
 
+// Plugins that are not in our official list can only be specified in
+// `netlify.toml` providing they are also installed in the site's package.json.
+// Otherwise, the build should fail. For backward compatibility, we only print
```

Comment:
Is `For backward compatibility, we only print a warning message at the moment.` correct?

Reply:
Right!